_posts/en/newsletters/2018-10-09-newsletter.md

```diff
@@ -0,0 +1,283 @@
+---
+title: 'Bitcoin Optech Newsletter #16: Scaling Bitcoin Special'
+permalink: /en/newsletters/2018/10/09/
+name: 2018-10-09-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter includes a summary of several notable talks
+presented at the Scaling Bitcoin workshop this past weekend.
+
+*Note:* this week's newsletter consists entirely of Scaling Bitcoin
+coverage as a result of there being almost nothing else to report in our
+usual Action Items, News, and Notable Code Changes sections.  We hope to
+return to our usual format next week as various Bitcoin infrastructure
+contributors start returning home from the workshop.
+
+## Workshop summary: Scaling Bitcoin V (Tokyo 2018)
+
+The fifth Scaling Bitcoin conference was held Saturday and Sunday in
+Tokyo, Japan.  In the sections below, we provide brief overviews to some
+of the talks we think might be most interesting to this newsletter's
+readers, but we also recommend watching the complete set of [videos][]
+provided by the workshop organizers or reading the [transcripts][]
+provided by Bryan Bishop.
+
+For convenience, at the end of each summary we directly link to its
+video and transcript (and paper, if available).  We also indicate
+whether its primary idea would require a *hard fork*, a *soft fork*, or
+is *possible now* if software implementing the idea is made available.
+Talks are listed below in the order they appeared in the workshop
+schedule.
+
+**Warning:** the following summaries may contain errors due to many of
+the talks describing subjects well beyond the expertise of the summary
+author.
+
+### Adjusting Bitcoin's block subsidy
+
+*Research by Anthony (AJ) Towns*
+
+This talk asks whether Bitcoin pays more for security than it needs to
```

Comment:
I think it's important to include AJ's introduction "And this idea is also not endorsed by me, and I would just like people to think about it." Suggesting changes to Bitcoin's monetary policy is indeed playing with fire, and I wouldn't want to lead the angry internet mob to AJ's door :)